sisl/geom/tests/test_geom.py

```diff
@@ -13,36 +14,59 @@
 pytestmark = [pytest.mark.geom]
 
 
+class CellDirect(SuperCell):
+
+    @property
+    def volume(self):
+        return dot3(self.cell[0, :], cross3(self.cell[1, :], self.cell[2, :]))
+
+
+def is_positive(geometry):
```

Comment:
Maybe this function could be named `is_righthanded` to be more explicit about what is tested?

Reply:
done! Thanks!